packages/components/organisms/f-checkout/src/components/Header.vue

```diff
@@ -0,0 +1,182 @@
+<template>
+    <div
+        data-test-id='header-component'
+        :class="$style['c-header']">
+        <div
+            v-if="isLoggedIn"
+            data-test-id='user-header'>
+            <h2 data-test-id='user-title'>
+                {{ $t('header.user.title', { name }) }}
+            </h2>
+
+            <p :class="$style['c-header-loginLink']">
+                <a
+                    :href="loginUrl"
+                    data-test-id="switch-user-link"
+                    @click="onVisitLoginPage">
+                    {{ $t('header.user.switchUser', { name }) }}
+                </a>
+            </p>
+        </div>
+
+        <div
+            v-else
+            data-test-id='guest-header'>
+            <h2 :class="$style['c-header-title']">
+                {{ $t('header.guest.guestTitle') }}
+            </h2>
+
+            <a
+                :href="loginUrl"
+                data-test-id="guest-login-button"
+                :class="$style['c-header-loginButton']"
+                @click="onVisitLoginPage">
+                {{ $t('header.guest.loginButton') }}
+            </a>
+
+            <div
+                :class="$style['c-header-option']">
+                <span>{{ $t('header.guest.option') }}</span>
+            </div>
+
+            <h2 :class="$style['c-header-title']">
+                {{ $t('header.guest.guestTitle') }}
+            </h2>
+
+            <p :class="$style['c-header-confirmation']">
+                {{ $t('header.guest.confirmation') }}
+            </p>
+        </div>
+    </div>
+</template>
+
+<script>
+import { mapState } from 'vuex';
+import EventNames from '../event-names';
+
+export default {
+    props: {
+        loginUrl: {
+            type: String,
+            required: true
+        }
+    },
+
+    computed: {
+        ...mapState('checkout', [
+            'customer',
+            'isLoggedIn'
+        ]),
+
+        name () {
+            return (this.customer.firstName.charAt(0).toUpperCase() + this.customer.firstName.slice(1));
+        }
+    },
+
+    methods: {
+        onVisitLoginPage () {
+            this.$emit(EventNames.CheckoutVisitLoginPage);
+        }
+    }
+};
+</script>
+
+<style lang="scss" module>
+.c-header {
```

Comment:
I'd name this to be specific to checkout (`c-checkoutHeader`) otherwise if these classes ever get exposed outside of this component in future for some reason they'll clash with the global site header classes.

Reply:
Or is this duplicating the global header?